### PR TITLE
Validating webhook

### LINKED ISSUE #19: Apply operator CR API overrides
- [x] spec.image.image
- [x] spec.image.imagePullPolicy
- [x] spec.audit.replicas
- [x] spec.audit.auditInterval
- [x] spec.audit.constraintViolationLimit
- [x] spec.audit.auditFromCache
- [x] spec.audit.auditChunkSize
- [x] spec.audit.logLevel
- [x] spec.audit.emitAuditEvents
- [ ] spec.validatingWebhook
- [x] spec.webhook.replicas
- [x] spec.webhook.logLevel
- [x] spec.webhook.emitAdmissionEvents
- [x] spec.webhook.failurePolicy
- [x] spec.nodeSelector
- [x] spec.affinity
- [x] spec.tolerations
- [x] spec.podAnnotations
- [x] spec.resources

### PR DESCRIPTION
**Pull request by ruromero**
Fix #19 missing item for configuring `validatingWebhook`

Depends on #50

### CODE REVIEW

**Review by JAORMX — APPROVED** [review]
nit: This can be done in a subsequent PR. Could you add docs stating that the default is to deploy the webhook? e.g. if `validatingWebhook` is left out, it'll deploy it by default.

Of course, another option is to set the default of that parameter explicitly.

You could do so by using:

`// +kubebuilder:default:=Enabled`

On the parameter as a comment.

**Comment by ruromero** [comment]
@JAORMX I've added both, let me know if you think it's accurate

File: api/v1alpha1/gatekeeper_types.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ type GatekeeperSpec struct {
 	Image *ImageConfig `json:"image,omitempty"`
 	// +optional
 	Audit *AuditConfig `json:"audit,omitempty"`
-	// +optional
+	// +kubebuilder:default:=Enabled
```
Comment (font) [inline api/v1alpha1/gatekeeper_types.go:37]: We should still leave the `+optional` marker to indicate that this field is optional to coincide with the `omitempty` json tag.

File: README.md [hunk 2]
```diff
@@ -108,4 +108,33 @@ If you would like to deploy Operator using OLM, you'll need to build and push th
...
+    ```
+
+## Usage 
+
+Before using Gatekeeper you have to create a `gatekeeper` resource that will be consumed by the operator and create all the necessary resources for you.
+
+Here you can find an example of a `gatekeeper` resource definition:
+
+```yaml
+```yaml
```
Comment (font) [inline README.md:120]: Duplicate line here.

**Review by font — COMMENTED** [review]
Thanks @ruromero !

**Review by font — APPROVED** [review]

**Pull request merged**